examples/to_mdbx.rs

```diff
@@ -0,0 +1,150 @@
+                growth_step: Some(1 * MB), // The growth step must be greater than zero to allow the database to grow, but also reasonable not too small, since increasing the size by little steps will result a large overhead.
+                shrink_threshold: Some(4 * MB), // The shrink threshold must be greater than zero to allow the database to shrink but also reasonable not too small (to avoid extra overhead) and not less than growth step to avoid up-and-down flouncing.
+            }),
+            ..DatabaseOptions::default()
+        },
+    )
+}
+
+fn read_tables_from(
+    path_to_bytecodes_file: impl AsRef<Path>,
+    path_to_pre_state_file: impl AsRef<Path>,
+    path_to_block_hash_file: Option<impl AsRef<Path>>,
+) -> Result<Tables, Error> {
+    let mut result = Tables::default();
+
+    let bytecodes: BTreeMap<B256, EvmCode> = {
+        let file = File::open(path_to_bytecodes_file)?;
+        bincode::deserialize_from(BufReader::new(file)).map_err(Error::other)?
+    };
+    for (code_hash, evm_code) in bytecodes {
+        let bytes = Bytecode::from(evm_code).original_bytes();
+        result.code_by_hash.insert(code_hash, bytes);
+    }
+
+    let state: HashMap<Address, EvmAccount> = {
+        let file = File::open(path_to_pre_state_file)?;
+        serde_json::from_reader(BufReader::new(file))?
+    };
+    for (address, account) in state {
+        assert!(account.code.is_none());
+        result.balance.insert(address, account.basic.balance.into());
+        result.nonce.insert(address, account.basic.nonce.into());
+        if let Some(code_hash) = account.code_hash {
+            result.code_hash.insert(address, code_hash);
+        }
+        if !account.storage.is_empty() {
+            result.has_storage.insert(address, B0::default());
+            for (index, storage_value) in account.storage {
+                result.storage.insert(
+                    B416::from_slice(&[address.as_slice(), B256::from(index).as_slice()].concat()),
+                    storage_value.into(),
+                );
+            }
+        }
+    }
+
+    if let Some(path) = path_to_block_hash_file {
+        let file = File::open(path)?;
+        let block_hashes: HashMap<U64, B256> = serde_json::from_reader(BufReader::new(file))?;
+        result.block_hash = block_hashes
+            .into_iter()
+            .map(|(k, v)| (B64::from(k), v))
+            .collect()
+    }
+
+    Ok(result)
+}
+
+fn write_table_to<E: DatabaseKind, K: AsRef<[u8]>, V: AsRef<[u8]>>(
+    db: &Database<E>,
+    table_name: &str,
+    entries: impl Iterator<Item = (K, V)>,
+) -> Result<(), libmdbx::Error> {
+    let tx = db.begin_rw_txn()?;
+    let table = tx.create_table(Some(table_name), TableFlags::default())?;
+    for (k, v) in entries {
+        tx.put(&table, k, v, WriteFlags::UPSERT)?;
+    }
+    tx.commit()?;
+    Ok(())
+}
+
+fn main() -> Result<(), Error> {
```

Comment:
This should be a simple function (with hard-coded paths) that:
1. Opens a database.
2. Reads and write accounts.
3. Reads and write bytecodes.
4. Reads and write block hashes.

Things like `Args` and `Tables` are unnecessary.